src/Exceptionless.Core/Jobs/EventPostsJob.cs

```diff
@@ -49,6 +51,10 @@ public class EventPostsJob : QueueJobBase<EventPost> {
             _organizationRepository = organizationRepository;
             _projectRepository = projectRepository;
             _jsonSerializerSettings = jsonSerializerSettings;
+            
+            _appOptions = appOptions;
+            _maximumEventPostFileSize = _appOptions.Value.MaximumEventPostSize + 1000;
```

Comment:
Why are we adding 1000 to this?

Reply:
It's what we were doing before. I think it was for a buffer / container overhead.

Reply:
Any idea why? Seems really freakin' strange to me. I could see us maybe multiplying by 1024 because we want the option value to be in megabytes or something, but that's not what we are doing.

Reply:
It was for the container overhead. I can add 1024 to it.